Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -721,6 +723,32 @@ static void webkit_web_context_class_init(WebKitWebContextClass* webContextClass
             G_TYPE_NONE, 1,
             WEBKIT_TYPE_AUTOMATION_SESSION);
 
+    /**
+     * WebKitWebContext::automation-finished:
```

Comment:
As I said I think this should be WebKitAutomationSession::closed for example, or even will-close would be more accurate

Reply:
Changed the new event to `WebKitAutomationSession::will-close`